hphp/hack/src/typing/typing_visibility.ml

```diff
@@ -159,16 +159,19 @@ let check_typedef_access ~use_pos ~in_signature env td =
     internal
```

Comment:
There's actually a lot more we could do to improve the error messages in this file. Functions like `visibility_error`, `is_protected_visible`, and `is_private_visible` could all be updated in the same way we've updated `is_visible_for_obj` and `is_visible_for_class`. I held off on doing that in this PR to make sure it's something we actually want and to keep this diff manageable.

Reply:
If this is something you're interested in working on in follow-up PRs, please do :)

I'd like us to avoid using "member" in all our error messages. It's less useful for users.

Reply:
Cool. I wonder if there's a way to do this more automatically instead of having to check in every function 😓